Oxford Reference.js

```diff
@@ -0,0 +1,226 @@
+	if(edition)
+		item.edition = edition;
+
+	var dateCreated = ZU.xpathText(doc, '//meta[@property="http://schema.org/dateCreated"]/@content');
+	if(dateCreated)
+		item.date = dateCreated;
+	else
+		item.date = ZU.xpathText(doc, '//dl[@class="metadata metadataPrintPublicationDate"]/dd');
+	
+	var isbn = ZU.xpathText(doc, '//meta[@property="http://schema.org/isbn"]/@content');
+	if(isbn)
+		item.ISBN = isbn;
+	else
+		item.ISBN = ZU.xpathText(doc, '//dl[@class="metadata metadataPrintIsbn13"]/dd');
+	
+	var publisher = ZU.xpathText(doc, '//meta[@property="http://schema.org/publisher"]/@content');
+	if(publisher)
+		item.publisher = publisher;
+	else
+		item.publisher = ZU.xpathText(doc, '//dl[@class="metadata metadataPublisher"]/dd');
+	
+
+	var title = ZU.xpathText(doc, '//meta[@property="og:title"]/@content');
+	if(title)
+		item.title = title;
+	else
+		item.title = ZU.xpathText(doc, '//h1[@id="pagetitle"]');
+	
+	var abstract = ZU.xpathText(doc, '//meta[@name="description"]/@content');
+	if(abstract)
+		item.abstractNote = abstract;
+	else
+		item.abstractNote = ZU.xpathText(doc, '//div[@class="abstract"]');
+	
+	var editors = ZU.xpath(doc, '//meta[@property="http://schema.org/editor"]/@content');
+	if(editors)
+	{
+		for (var i in editors){
+			item.creators.push(ZU.cleanAuthor(editors[i].textContent,"editor"));
+		}
+	}
+	
+	var authors = ZU.xpathText(doc,'//dl[@class="metaInfo"]/dd[@class="author"]');
+	if(authors)
+	{
+		authors = authors.split(',');
+		for (var i in authors){
+			item.creators.push(ZU.cleanAuthor(authors[i], "author"));
+		}
+		
+	}
+	
+	item.attachments = ({
+		url: url,
+		title: "Oxford Reference Snapshot",
+		mimeType: "text/html"
+	});
+	
+	item.complete();	
+}/** BEGIN TEST CASES **/
+var testCases = [
+	{
+		"type": "web",
+		"url": "http://www.oxfordreference.com/view/10.1093/acref/9780199546572.001.0001/acref-9780199546572-e-0009",
+		"items": [
+			{
+				"itemType": "bookSection",
+				"title": "Accutane - Oxford Reference",
+				"creators": [
+					{
+						"firstName": "Andrew",
+						"lastName": "Hodges",
+						"creatorType": "author"
+					}
+				],
+				"date": "2008",
+				"ISBN": "9780199546572",
+				"abstractNote": "Isotretinoin. The synthetic retinoid derivative 13-cis-retinoic acid (Accutane) used for severe Acne vulgaris. The dose is 1",
+				"libraryCatalog": "Oxford Reference",
+				"publisher": "Oxford University Press",
+				"attachments": {
+					"url": "http://www.oxfordreference.com/view/10.1093/acref/9780199546572.001.0001/acref-9780199546572-e-0009",
+					"title": "Oxford Reference Snapshot",
+					"mimeType": "text/html"
+				},
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "http://www.oxfordreference.com/view/10.1093/acref/9780199608218.001.0001/acref-9780199608218",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Concise Oxford Companion to English Literature - Oxford Reference",
```

Comment:
We should clean the title such that the output is without the " - Oxford Reference".